packages/seacas/libraries/ioss/src/Ioss_Utils.C

```diff
@@ -73,41 +73,71 @@ namespace {
-  Kokkos::View<int64_t *> data_view_int64;
-  Kokkos::View<double *>  data_view_double;
-  // Kokkos::View<Kokkos_Complex *> data_view_complex cannot be a global variable,
-  // Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
-  // have type std::complex entities.
-  Kokkos::View<char **>    data_view_2D_char;
-  Kokkos::View<int **>     data_view_2D_int;
-  Kokkos::View<int64_t **> data_view_2D_int64;
-  Kokkos::View<double **>  data_view_2D_double;
-  // Kokkos::View<Kokkos_Complex **> data_view_2D_complex cannot be a global variable,
-  // Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
-  // have type std::complex entities.
-  Kokkos::View<char **, Kokkos::LayoutRight, Kokkos::HostSpace>    data_view_2D_char_layout_space;
-  Kokkos::View<int **, Kokkos::LayoutRight, Kokkos::HostSpace>     data_view_2D_int_layout_space;
-  Kokkos::View<int64_t **, Kokkos::LayoutRight, Kokkos::HostSpace> data_view_2D_int64_layout_space;
-  Kokkos::View<double **, Kokkos::LayoutRight, Kokkos::HostSpace>  data_view_2D_double_layout_space;
-// Kokkos::View<Kokkos_Complex **, Kokkos::LayoutRight, Kokkos::HostSpace>
-// data_view_2D_complex_layout_space cannot be a global variable,
-// Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
-// have type std::complex entities.
+    Kokkos::View<char *>    data_view_char;
+    Kokkos::View<int *>     data_view_int;
+    Kokkos::View<int64_t *> data_view_int64;
+    Kokkos::View<double *>  data_view_double;
+    // Kokkos::View<Kokkos_Complex *> data_view_complex cannot be a global variable,
+    // Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
+    // have type std::complex entities.
+    Kokkos::View<char **>    data_view_2D_char;
+    Kokkos::View<int **>     data_view_2D_int;
+    Kokkos::View<int64_t **> data_view_2D_int64;
+    Kokkos::View<double **>  data_view_2D_double;
+    // Kokkos::View<Kokkos_Complex **> data_view_2D_complex cannot be a global variable,
+    // Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
+    // have type std::complex entities.
+    Kokkos::View<char **, Kokkos::LayoutRight, Kokkos::HostSpace> data_view_2D_char_layout_space;
+    Kokkos::View<int **, Kokkos::LayoutRight, Kokkos::HostSpace>  data_view_2D_int_layout_space;
+    Kokkos::View<int64_t **, Kokkos::LayoutRight, Kokkos::HostSpace>
+        data_view_2D_int64_layout_space;
+    Kokkos::View<double **, Kokkos::LayoutRight, Kokkos::HostSpace>
+        data_view_2D_double_layout_space;
+    // Kokkos::View<Kokkos_Complex **, Kokkos::LayoutRight, Kokkos::HostSpace>
+    // data_view_2D_complex_layout_space cannot be a global variable,
+    // Since Kokkos::initialize() has not yet been called. Also, a Kokkos:View cannot
+    // have type std::complex entities.
+#endif
+    DataPool()
+    {
+#ifdef SEACAS_HAVE_KOKKOS
+  data_view_char      = Kokkos::View<char *>("view_char", 0);
```

Comment:
These constructor invocations are harmless but unnecessary.  Kokkos::View's default constructor initializes the View to have all zero dimensions.

Reply:
Thanks.  I will remove these then.